components/style/properties/shorthand/font.mako.rs

```diff
@@ -50,7 +51,11 @@
             if let Ok(sys) = input.try(SystemFont::parse) {
                 return Ok(expanded! {
                      % for name in SYSTEM_FONT_LONGHANDS:
-                         ${name}: ${name}::SpecifiedValue::system_font(sys),
+                         % if name == "font_size":
+                             ${name}: FontSize::system_font(sys),
+                         % else:
+                             ${name}: ${name}::SpecifiedValue::system_font(sys),
+                         % endif
```

Comment:
I saw it got [error](https://travis-ci.org/servo/servo/jobs/298879594) from this line in Travis.
Due to unfamiliar with the `SYSTEM_FONT_LONGHANDS`, I just tried to change it with this ugly solution.
If there's any better solution, please tell me and I'll update it.

Reply:
Yeah, this is fine, I can't think of anything better right now. This system_font thing is quite messy too...

Reply:
Thanks for review!